examples/mayavi/data_interaction/normal_flipping_stl.py

```diff
@@ -0,0 +1,101 @@
+"""
+This example shows how to read a stl file and flip it's normals.
+
+The example stl file can be downloaded from
+https://cults3d.com/en/3d-model/various/pyramide-carree
+
+"""
+
+import vtk
+import numpy as np
+from vtk.util.numpy_support import vtk_to_numpy
+from mayavi import mlab
+
+# reading the stl file and getting the triangle data
+reader = vtk.vtkSTLReader()
+reader.SetFileName("pyramide4.stl")
```

Comment:
The example should be self-contained, so I would suggest making your own stl file or even perhaps allowing the user to pass one in as a command line argument.  And default to a simple one if no argument is passed.